frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -309,35 +318,42 @@ function ReaderDictionary:addToMainMenu(menu_items)
                 hold_callback = function(touchmenu_instance)
                     self:toggleFuzzyDefault(touchmenu_instance)
                 end,
-                separator = true,
             },
             {
-                text = _("Enable dictionary lookup history"),
+                text = _("Dictionary lookup history"),
```

Comment:
Not sure Fuzzy search and Dict lookup history should be in the same separator'ed area :)
I would leave lookup history alone with separator above and below,